gui_qt/app.py

```
@@ -1052,6 +1066,129 @@ def _build_workbench_tab(self) -> None:
+        rag_row.addWidget(self.context_rag_status_label, 1)
+        self.context_bootstrap_rag_btn = QPushButton("预建记忆库")
+        self.context_bootstrap_rag_btn.setObjectName("secondary_btn")
+        self.context_bootstrap_rag_btn.clicked.connect(
+            lambda: self._on_context_bootstrap_clicked("rag")
+        )
+        rag_row.addWidget(self.context_bootstrap_rag_btn, 0)
+        outer.addLayout(rag_row)
+
+        idx_row = QHBoxLayout()
+        idx_row.setSpacing(8)
+        self.context_source_index_status_label = QLabel("原文索引：—")
```

Ruff (0.15.20)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)